.chloggen/36650.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

```suggestion
change_type: enhancement
```
looks like an enhancement to me

---

resolved.